### DISCUSSION TITLE
Broken layout in merge request view

### DISCUSSION THREAD

**Discussion by Blaisorblade** [community/community#36587] [Pull Requests]
The layout in https://github.com/VeriNum/vcfloat/pull/5/files/f13a3ddff7c41eff8990ee38074e7d74010aefdc..3ef16ce371bb956c65cc1ee744ebdd0ff536349c is currently broken, compared to the expected one (used correctly in https://github.com/VeriNum/vcfloat/pull/5/files).

Broken:
![image](https://user-images.githubusercontent.com/289960/196523333-b258d5fc-3763-43db-9897-de6377ab5df6.png)

Expected:
![image](https://user-images.githubusercontent.com/289960/196523375-283d1cc2-4391-439c-bdd6-83d8f2376468.png)

**Comment by ossanna16** [comment]
Hey there @Blaisorblade and thank you for the feedback! I'm glad you found our community 🙂
